labs/lab-2-javascript-game/script.js

```diff
@@ -0,0 +1,43 @@
+var playerHealth = 40;
+var grantHealth = 10;
+var winCount = 0;
+
+function getDamage() {
+    damage = Math.floor(Math.random() * 5) + 1
+    return damage
+}
+
+function startCombat(damage) {
+   damage = getDamage();
+   console.log("You took ", damage, "damage!");
+   playerHealth = playerHealth - damage;
+   console.log("You have health ", playerHealth, "left.");
+
+  damage = getDamage();
+
+   console.log("Grant took ", damage, "damage!");
+   grantHealth = grantHealth - damage;
+   console.log("Grant has health ", grantHealth, "left.");
+}
+
+
+function startGame() {
+  var play = prompt("Would you like to play?");
+
+  if (play === "Yes") {
+    var playerName = prompt("What is your name?");
+    while (play === "Yes") {
```

Comment:
If play never changes inside this loop, you'll play FOREVER. Good for a horror movie, bad for a program.

Think about what condition actually belongs here; you want to stop the game:

- if the player says "quit"
- if the player dies
- or if the player wins